resources/privacy-and-gdpr.md

```diff
@@ -61,13 +60,45 @@ To programmatically set the privacy status for the application user:
 #### Syntax
 
 ```java
-public static void setPrivacyStatus(final MobilePrivacyStatus privacyStatus);
+public static void update(final Map<String, Object> xdmFormattedConsents);
 ```
 
 #### Example
 
 ```java
-MobileCore.setPrivacyStatus(MobilePrivacyStatus.OPT_OUT);
+// example 1, updating users collect consent to 'yes'
+final Map<String, Object> consents = new HashMap<String, Object>() {
+	{
+		put("consents", new HashMap<String, Object>() {
+			{
+				put("collect", new HashMap<String, String>() {
+					{
+						put("val", "y");
+					}
+				});
+			}
+		});
+	}
+};
+
+Consent.update(consents);
+
+// example 2, updating users collect consent to 'no'
+final Map<String, Object> consents = new HashMap<String, Object>() {
+	{
+		put("consents", new HashMap<String, Object>() {
+			{
+				put("collect", new HashMap<String, String>() {
+					{
+						put("val", "n");
+					}
+				});
+			}
+		});
+	}
+};
+
+Consent.update(consents);
```

Comment:
let's add an info note here

After a user has selected collect consent no (n), the SDK will not allow you to set the users collect consent to yes (y).